go.mod

```diff
@@ -171,3 +171,5 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+go 1.13
```

Comment:
I don't think we should change this. This would force everyone to upgrade to Go 1.13. This is really another topic altogether.

Reply:
Yeah, we would need some more changes to make this work reliably.

Reply:
Removed. This was a result of the wrong version of Go installed on my dev machine